src/Schema.php

```diff
@@ -29,6 +29,9 @@ public static function activate() {
 
     // Force a long client-side caching time for assets with "ver" query string.
     static::ensureAssetsCacheControl();
+
+    // Includes the necessary response headers for all requests in htaccess file.
```

Comment:
Code documentation should be more concise, summarizing what is being done and for what reason. "Including headers" does not tell the reader much, as headers are used for a lot of different things.
```suggestion
    // Ensures security related HTTP headers in all responses to prevent clickjacking,
    // XSS, and other vulnerabilities.
```

Reply:
Well that's the point, you are being too much specific of the current purpose of this new feature, when in general terms it can be used for a wider scope, meaning all kind of headers we may need in the future, besides you also had already another module/chunk called  just "security", so I'd like to keep it a bit "open to extension" here as SOLID states

Reply:
I would not advise to mix non-security related headers into this block. If a feature comes up that needs further HTTP response headers, it can introduce a new block.

You're right; there is already a module named security and it should have been named better. It predates our current practice of organizing one feature into one class (instead of grouping them by topic/domain), to yield a better overview of what functionality is affected or introduced by a plugin.